tests/test_assembly_input.nf.test

```diff
@@ -15,6 +15,63 @@ nextflow_pipeline {
         }
 
         then {
+            // Output dir: Assembly
+            def stable_name_assembly = getAllFilesFromDir(
+                params.outdir, include: ['Assembly/**'], relative: true, includeDir: false
+            )
+            def bowtie2_logs = getAllFilesFromDir(
+                params.outdir, include: ['Assembly/{SPAdes,MEGAHIT}/QC/*/*.bowtie2.log']
+            )
+
+            // Output dir: Annotation
+            def stable_name_annotation = getAllFilesFromDir(
+                params.outdir, include: ['Annotation/**'], relative: true, includeDir: false
+            )
+            def stable_path_annotation = getAllFilesFromDir(
+                params.outdir, include: ['Annotation/**'], ignoreFile: 'tests/.nftignore'
+            )
+
+            // Output directory: GenomeBinning
+            def stable_name_binning = getAllFilesFromDir(
+                params.outdir,
+                include: ['GenomeBinning/**'],
+                ignore: ['GenomeBinning/QC/BUSCO/**/{logs,prodigal_output,busco_sequences}/**'],
```

Comment:
doesnt that overlap with .nftignore? Wouldnt it be better to use that file instead?